arm-datalake-analytics/catalog/2016-11-01/swagger/catalog.json

```diff
@@ -374,6 +370,90 @@
         }
       }
     },
+    "/catalog/usql/databases/{databaseName}/credentials": {
+      "get": {
+        "tags": [
+          "Catalog"
+        ],
+        "operationId": "Catalog_ListCredentials",
+        "description": "Retrieves the list of credentials from the Data Lake Analytics catalog.",
+        "parameters": [
+          {
+            "name": "databaseName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the database containing the schema."
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "OData filter. Optional."
+          },
+          {
+            "name": "$top",
+            "in": "query",
+            "required": false,
+            "type": "integer",
```

Comment:
No max but I will set the min to 1, since anything less than that does not make sense. We will page for anything more than our max page count.